code/game/objects/machinery/computer/communications.dm

```diff
@@ -203,10 +203,10 @@
 				var/Ship[] = SSticker.mode.count_humans_and_xenos(SSmapping.levels_by_trait(ZTRAIT_MARINE_MAIN_SHIP))
 				var/ShipMarines[] = Ship[1]
 				var/ShipXenos[] = Ship[2]
-				var/Planet[] = SSticker.mode.count_humans_and_xenos(SSmapping.levels_by_trait(ZTRAIT_GROUND))
-				var/PlanetMarines[] = Planet[1]
-				var/PlanetXenos[] = Planet[2]
-				if((PlanetXenos < round(PlanetMarines * 0.8)) && (ShipXenos < round(ShipMarines * 0.5))) //If there's less humans (weighted) than xenos, humans get home-turf advantage
+				var/All[] = SSticker.mode.count_humans_and_xenos()
+				var/AllMarines[] = All[1]
+				var/AllXenos[] = All[2]
+				if((AllXenos < round(AllMarines * 0.8)) && (ShipXenos < round(ShipMarines * 0.5))) //If there's less humans (weighted) than xenos, humans get home-turf advantage
 					log_game("[key_name(usr)] has attemped to call a distress beacon, but it was denied due to lack of threat.")
```

Comment:
This isn't really necessary anymore

Reply:
Is it because its covered in the following line?

Reply:
Yes, this was used as a debug measure back in the CM code days and used to be much more spammy. There's no need to log this now.